test/manifests/fakerp/admin-update.yaml

```diff
@@ -1,3 +1,6 @@
 # ADMIN_MANIFEST=test/manifests/fakerp/admin-update.yaml hack/upgrade.sh
 properties:
   clusterVersion: latest
+  monitorProfile:
```

Comment:
I do not believe this file should be being changed.  What is the rationale for changing it?